cypress/integration/lists/status-list.spec.js

```diff
@@ -0,0 +1,33 @@
+/// <reference types="cypress" />
+
+// todo add data-test=id to info list items
+describe('Functional status list loads & tag', () => {
+    beforeEach(() => {
+        cy.visit('http://localhost:3000/status-list');
+    });
+
+    it('title displays', () => {
+        cy.get('[data-cy=pxb-toolbar]').should('contain', 'Status List');
+    });
+
+    it('status tags load', () => {
+        cy.get('[data-test=list-item-tag]').should('be.visible').and('have.css', 'background-color', 'rgb(202, 60, 61)')

```

Comment:
Won't this fail if the randomly generated data does not contain an alarm?

Reply:
Not much here to do, removed tests.

Reply:
Yeah, these randomly generated datasets are quickly losing my favor....makes it very challenging to test. I remember when Chinna was trying to do snapshot tests for these and every time code got checked in they would break :)